model/task_lifecycle_test.go

```diff
@@ -1190,6 +1191,66 @@ func TestAbortTask(t *testing.T) {
 	})
 
 }
+func TestTryDequeueAndAbortCommitQueueVersionWithBlacklist(t *testing.T) {
+	assert.NoError(t, db.ClearCollections(patch.Collection, VersionCollection, task.Collection, build.Collection, commitqueue.Collection))
+	patchID := mgobson.NewObjectId().Hex()
+	v := &Version{
+		Id:     patchID,
+		Status: evergreen.VersionStarted,
+	}
+
+	p := &patch.Patch{
+		Id:          patch.NewId(patchID),
```

Comment:
potentially we could insert a patch in the db without an id and let the driver/db decide the id.
The version's ID could be something else. No?

Reply:
I don't think that works.

Reply:
😄 
That makes sense